src/utils.hpp

```diff
@@ -90,6 +91,11 @@ inline bool is_scalar(const ddwaf_object *obj)
     return obj != nullptr && (obj->type & PWI_DATA_TYPES) != 0;
 }
 
+inline bool is_invalid_or_null(const ddwaf_object *obj)
+{
+    return obj != nullptr && obj->type == DDWAF_OBJ_INVALID && obj->type == DDWAF_OBJ_NULL;
```

Comment:
you mean `obj != nullptr && (obj->type == DDWAF_OBJ_INVALID || obj->type == DDWAF_OBJ_NULL)` I guess

Reply:
Ah yes, I changed it without thinking...